core/examples/luigi-example-next/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "luigi-example-next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
```

Comment:
I am newly getting an error when first trying to run the app after `npm install`. This didn't occur before . 
`Repro steps: `Re-clone the repo and `npm install` then `npm run dev`

Error log: 
```
$ npm run dev

> luigi-example-next@0.1.0 dev /luigi/core/examples/luigi-example-next
> next dev

ready - started server on http://localhost:3000
Error: Cannot find module 'postcss-preset-env' from '/luigi/core/examples/luigi-example-next/'
```

Let me know if you could reproduce it. 

I did install that missing module afterwards to make it work, but that seemed like a weird fix, also since others didn't get this error probably worth investigating a bit if it occurs.

Reply:
I also did npm install and then npm run dev and all worked well for me. just fyi

Reply:
Well... I didn't face this issue. Not hear other reviewers bring it up either. @UlianaMunich @maxmarkus , did you guys face this problem?

Reply:
If @maxmarkus  also doesn't get it, I assume then it should be good to go 👍  🟢  since I'm getting weird behaviours from other PR's recently too, probably from changing npm or node versions, so could be something only happening in my machine.

Reply:
Seems to be resolved. Something local on my side